tests/utils/mod.rs

```diff
@@ -247,3 +247,10 @@ impl GetSharedClaims for UnregisterIdentityRequestAuth {
         &self.shared_claims
     }
 }
+
+pub struct MockGetRpcUrl;
+impl GetRpcUrl for MockGetRpcUrl {
+    fn get_rpc_url(&self, _: String) -> Option<Url> {
+        None
```

Comment:
Does this work in tests as it's always `None`?

Reply:
EIP-1271 is not tested by Notify Server tests (however it should be in the future). So since this function will never be called, it's ok to return `None`.